src/atomics/poly.jl

```diff
@@ -1,27 +1,39 @@
-
-function poly(scene::makie, points, attributes::Dict)
-    last(points) == first(points) && pop!(points)
-    polys = GeometryTypes.split_intersections(points)
-    result = []
-    for poly in polys
-        mesh = GLNormalMesh(poly) # make polygon
-        if !isempty(GeometryTypes.faces(mesh)) # check if polygonation has any faces
-            push!(result, GLVisualize.visualize(mesh, Style(:default), kw_args))
-        else
-            warn("Couldn't draw the polygon: $points")
-        end
+function _poly(scene::makie, attributes::Dict)
+    attributes = poly_defaults(scene, attributes)
+    println(typeof(attributes[:positions]))
+    bigmesh = lift_node(attributes[:positions]) do p
+        polys = GeometryTypes.split_intersections(p)
+        merge(GLPlainMesh.(polys))
```

Comment:
What's the `merge` doing here? This is from `GeometryTypes/src/meshes.jl`, right? Does this basically put multiple meshes into a single structure?